duckduckgo/iOS
# design fixes (fonts, icons, cookie management banner)

Task/Issue URL: https://app.asana.com/0/414235014887631/1204701159832274/f

and:
* https://app.asana.com/0/414235014887631/1204807921061412/f

Tech Design URL:
CC:

**Description**:

* Remove all references to Proxima Nova in storyboards and xibs
* Updated fonts to set their expected weight using the system font
* Fixed dax logo alignment in address bar
* Fixed dax logo alignment when onboarding
* Fixed cookie management prompt

**Steps to test this PR**:

1. Smoke test the whole app including:
* onboarding
* settings
* address bar
* browsing menu
* anything else you can think of

2. Make sure onboarding animation logo transition looks good on a range of devices

3. Make sure the DDG logo is centre aligned in the address bar



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
